File: test/test-assign/r-correct/autograder/tests/q1.R
test = list(
  name = "q1",
  cases = list(
    ottr::TestCase$new(
      hidden = FALSE,
      name = NA,
      points = 1,
      code = {
        
      }
    ),
    ottr::TestCase$new(
      hidden = FALSE,
      name = NA,
      points = 1,
      code = {
        testthat::expect_true(x < 100)
      }
    ),
    ottr::TestCase$new(
      hidden = TRUE,
      name = NA,
      points = 1,
      code = {
        testthat::expect_equal(x, 2)
      }
    ),
    ottr::TestCase$new(
      hidden = TRUE,
      name = NA,
      points = 2,
      code = {
        testthat::expect_equal(as.character(x), "2")
      }
    )
  )
)

File: test/test_check/files/tests/q1.py
OK_FORMAT = True

test = {
	"name": "q1",
	"points": 1,
	"suites": [ 
		{
			"cases": [ 
				{
					"code": r"""
					>>> 1 == 1
					True
					""",
					"hidden": False,
					"locked": False,
				}, 
				{
					"code": r"""
					>>> 1 == 2
					False
					""",
					"hidden": False,
					"locked": False,
				}, 
			],
			"scored": False,
			"setup": "",
			"teardown": "",
			"type": "doctest"
		}, 
	]
}
